electron/utils.ts

```diff
@@ -326,7 +328,11 @@ function resetHeroic() {
 function showItemInFolder(item: string) {
   if (existsSync(item)) {
     try {
-      shell.showItemInFolder(item)
+      if (isWindows) {
+        execAsync(`explorer /Select,"${item}"`)
```

Comment:
`shell.showItemInFolder` doesnt work on windows?

Reply:
It didn't work for me, and @Laria (can't tag properly on mobile) also had the same issue

Reply:
Hm ok then.
I forgot to test this yesterday when I was on windows.